rtgui/paramsedited.cc

```diff
@@ -1403,18 +1403,19 @@ void ParamsEdited::combine (rtengine::procparams::ProcParams& toEdit, const rten
     if (localContrast.enabled) {
         toEdit.localContrast.enabled = mods.localContrast.enabled;
     }
-    if (localContrast.radius) {
-        toEdit.localContrast.radius = mods.localContrast.radius;
-    }
-    if (localContrast.amount) {
-        toEdit.localContrast.amount = mods.localContrast.amount;
-    }
-    if (localContrast.darkness) {
-        toEdit.localContrast.darkness = mods.localContrast.darkness;
-    }
-    if (localContrast.lightness) {
-        toEdit.localContrast.lightness = mods.localContrast.lightness;
-    }
+
+#define ADDSETVAL_(v, i)                                                   \
```

Comment:
Are the trailing backslashes `\` needed?

Reply:
of course they are